demo/greeting/src/main/scala/greeting/GreetingClientApp.scala

```diff
@@ -17,33 +17,57 @@
 package freestyle.rpc.demo
 package greeting
 
-import freestyle.rpc.demo.echo.EchoServiceGrpc
-import freestyle.rpc.demo.echo.EchoServiceGrpc.EchoServiceStub
-import freestyle.rpc.demo.echo_messages.EchoRequest
-import io.grpc.ManagedChannelBuilder
-
-import scala.concurrent.Await
+import cats._
+import cats.implicits._
+import freestyle._
+import freestyle.implicits._
+import freestyle.rpc.demo.echo_messages._
+import runtime.implicits.client._
+import greeting.client._
+import io.grpc._
+
+import scala.concurrent.{Await, Future}
 import scala.concurrent.duration.Duration
 
+@module
+trait ClientAPP {
+  val greetingClientM: GreetingClientM
+  val echoClientM: EchoClientM
+}
+
 object GreetingClientApp {
 
-  def main(args: Array[String]): Unit = {
+  val messageRequest = MessageRequest("Freestyle")
+  val echoRequest    = EchoRequest("echo...")
 
-    val request = MessageRequest("Freestyle")
-    val client  = new GreetingClient(host, portNode1)
-    // val client  = new GreetingClient(host, portNode2)
+  def unaryDemo[F[_]](implicit APP: ClientAPP[F]): FreeS[F, (String, String, String)] = {
+
+    val greetingClientM: GreetingClientM[F] = APP.greetingClientM
+    val echoClientM: EchoClientM[F]         = APP.echoClientM
+
+    val defaultOptions = CallOptions.DEFAULT
+
+    for {
+      hi   <- greetingClientM.sayHello(messageRequest, defaultOptions)
```

Comment:
Since these operations are not interdependent this could be better expressed in applicative style.
```
(a |@| b |@| c).map( ... )
```